src/components/common/ConnectWallet/ConnectButton.tsx

```diff
@@ -12,10 +11,8 @@ export const ConnectButton = () => {
   }
 
   return (
-    <Box className={css.buttonContainer}>
```

Comment:
I think we need the button container so that the wallet icon and address are on the same line

Reply:
Ugh, thanks for catching this. Fixed.